node/pkg/watchers/evm/watcher.go

```diff
@@ -151,9 +152,15 @@ func NewEthWatcher(
 func (w *Watcher) Run(ctx context.Context) error {
 	logger := supervisor.Logger(ctx)
 
+	useFinalizedBlocks := (w.chainID == vaa.ChainIDEthereum && (!w.unsafeDevMode))
 	if w.shouldCheckSafeMode {
 		if err := w.checkForSafeMode(ctx); err != nil {
-			return err
+			if !strings.Contains(err.Error(), "is not using safe mode") {
```

Comment:
Since `checkForSafeMode` is a function we control I think we should have it return `(bool, error)` rather than returning the value in-band.

Reply:
Changed.